rllib/evaluation/env_runner_v2.py

```diff
@@ -55,15 +55,35 @@
 class _PerfStats:
     """Sampler perf stats that will be included in rollout metrics."""
 
-    def __init__(self):
+    def __init__(self, ema=False, coeff=0.001):
+        # EMA mode.
```

Comment:
Can we write out the meaning of "EMA" here as well?

Reply:
done